canonical-web-and-design/charmhub.io
# Add dummy data to python

## Done

- Add dummy data to python and use it in templates
- Add 500 and 404 pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Homepage and `/charm` page should look the same but with some data come from the dummy data in python


## Issue / Card

Fixes #36
